packages/framework/aqueduct/src/container-runtime-factories/baseContainerRuntimeFactory.ts

```diff
@@ -46,17 +52,24 @@ export class BaseContainerRuntimeFactory
 	 * @param initializeEntryPoint - Function that will initialize the entryPoint of the ContainerRuntime instances
 	 * created with this factory
 	 */
-	constructor(
-		private readonly registryEntries: NamedFluidDataStoreRegistryEntries,
-		private readonly dependencyContainer?: IFluidDependencySynthesizer,
-		private readonly requestHandlers: RuntimeRequestHandler[] = [],
-		private readonly runtimeOptions?: IContainerRuntimeOptions,
-		private readonly initializeEntryPoint?: (
-			runtime: IContainerRuntime,
-		) => Promise<FluidObject>,
-	) {
+	constructor(props: {
+		registryEntries: NamedFluidDataStoreRegistryEntries;
+		dependencyContainer?: IFluidDependencySynthesizer;
+		requestHandlers?: RuntimeRequestHandler[];
+		runtimeOptions?: IContainerRuntimeOptions;
+		initializeEntryPoint: (runtime: IContainerRuntime) => Promise<FluidObject>;
+	}) {
```

Comment:
This constructor change probably wants its own changeset. I think I'd do a single changeset for this, `ContainerRuntimeFactoryWithDefaultDataStore`, `DOProviderContainerRuntimeFactory`, and `RuntimeFactory`,s constructor changes.